atomic

```diff
@@ -514,6 +518,18 @@ def create_parser(help_text):
                      help=_("Only display container IDs"))
     add_opt(pss)
 
+    # atomic sign
+    signer = None if ATOMIC_CONFIG['default_signer'] is None else ATOMIC_CONFIG['default_signer']
+    sig_store = "/var/lib/atomic/containers" if ATOMIC_CONFIG['default-sigstore-path'] is None else ATOMIC_CONFIG['default-sigstore-path']
+    signp = subparser.add_parser("sign", help="Sign an image",
+                                 epilog="Create a signature for an image which can be used later to verify it.")
+    signp.set_defaults(_class=Sign, func="sign")
+    signp.add_argument("images", nargs="*", help=_("images to sign"))
+    signp.add_argument("--sign_key", dest="sign_key", default=signer,
+                       help=_("Name of the signing key. A default can be defined in /etc/atomic.conf"))
```

Comment:
help=_("Name of the signing key. Currently %s, default can be defined in /etc/atomic.conf" % signer))